Offline/OfflinePlayer/objc/OfflinePlayer/VideosViewController.m

```diff
@@ -0,0 +1,1370 @@
+      kBCOVOfflineVideoManagerAllowsCellularPlaybackKey: @(NO),
+      kBCOVOfflineVideoManagerAllowsCellularAnalyticsKey: @(NO)
+      };
+    [BCOVOfflineVideoManager initializeOfflineVideoManagerWithDelegate:self
+                                                               options:optionsDictionary];
+    self.offlineVideoManager = BCOVOfflineVideoManager.sharedManager;
+    gVideosViewController = self;
+
+    NSLog(@"Using Brightcove Native Player SDK version %@", BCOVPlayerSDKManager.version);
+    
+    self.tabBarController = (UITabBarController*)self.parentViewController;
+    
+    self.videosTableView.dataSource = self;
+    self.videosTableView.delegate = self;
+    [self.videosTableView setContentInset:UIEdgeInsetsMake(0, 0, 8, 0)];
+ 
+    // Add a refresh control to the table view
+    {
+        self.refreshControl = [[UIRefreshControl alloc] init];
+        [self.refreshControl addTarget:self action:@selector(handleTableRefresh:) forControlEvents:UIControlEventValueChanged];
+        
+        if (floor(NSFoundationVersionNumber) <= NSFoundationVersionNumber_iOS_9_x_Max)
+        {
+            // iOS 9 and earlier: background view is the refresh control
+            self.videosTableView.backgroundView = self.refreshControl;
+        }
+        else
+        {
+            // iOS 10 and later: proper refresh control support
+            self.videosTableView.refreshControl = self.refreshControl;
+        }
+    }
+
+    [NSNotificationCenter.defaultCenter addObserverForName:kBCOVOfflineVideoManagerAnalyticsStorageFullWarningNotification
+                                                    object:nil
+                                                     queue:nil
+                                                usingBlock:^(NSNotification * _Nonnull note) {
+                                                    
+                                                    UIAlertController* alert = [UIAlertController alertControllerWithTitle:@"Analytics storage is full"
+                                                                                                                   message:@"Encourage the app user to go online"
+                                                                                                            preferredStyle:UIAlertControllerStyleAlert];
+                                                    UIAlertAction* defaultAction = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault
+                                                                                                          handler:^(UIAlertAction * action) {}];
+                                                    [alert addAction:defaultAction];
+                                                    [self presentViewController:alert animated:YES completion:nil];
+
+                                                }];
+    
+    [NSURLCache.sharedURLCache removeAllCachedResponses];
```

Comment:
We can remove this line - it looks suspicious. Was just used for testing.

Reply:
√